shared/js/background/broken-site-report.js

```diff
@@ -179,6 +180,17 @@ export async function breakageReportForTab ({
         }
     }
 
+    // collect page parameters
+    if (pageParams.docReferrer) {
+        if (pageParams.docReferrer.hostname === 'duckduckgo.com') {
+            tab.openerContext = 'serp'
+        } else {
+            tab.openerContext = 'navigation'
+        }
+    } else if (!pageParams.opener) {
+        tab.openerContext = 'external'
+    }
```

Comment:
Under what conditions do we actually use the referrer from the header? It looks like we just overwrite it with the document referrer here?